src/streams/WritableStream.ts

```diff
@@ -104,7 +112,7 @@ export default class WritableStream<T> {
 	protected _queue: SizeQueue<Record<T>>;
 	protected _writing: boolean;
 
-	constructor(underlyingSink: Sink<T>, strategy: Strategy<T> = {}) {
+	constructor(underlyingSink: Sink<T> = {}, strategy: Strategy<T> = {}) {
```

Comment:
Why did you add a default value for underlyingSink?  The code is written to handle an undefined underlyingSink.  It uses util.invokeOrNoop to take care of that.  Was this change made for some scenario that isWriteableStream needs to return true when a WritableStream is instantiated with no sink?  On line 210, the sink is set to undefined which is not consistent with it having a default value.